remme/shared/forms/_validators.py

```diff
@@ -21,3 +21,36 @@ def __call__(self, form, field):
 
             field.errors[:] = []
             raise validators.StopValidation(message)
+
+
+class DataRequired(object):
+    """
+    Checks the field's data is 'truthy' otherwise stops the validation chain.
+    This validator checks that the ``data`` attribute on the field is a 'true'
+    value (effectively, it does ``if field.data``.) Furthermore, if the data
+    is a string type, a string containing only whitespace characters is
+    considered false.
+    If the data is empty, also removes prior errors (such as processing errors)
+    from the field.
+    Args:
+    message (str): error message to raise in case of a validation error.
+    """
+    field_flags = ('required',)
+
+    def __init__(self, message=None):
+        self.message = message
+
+    def __call__(self, form, field):
+
+        if field.data == 0:
```

Comment:
Leave a comment to the documentation why you pass when data is zero.

Reply:
@dmytrostriletskyi, done.